packages/mongo/package.js

```diff
@@ -21,7 +21,7 @@ Npm.strip({
 });
 
 Package.onUse(function (api) {
-  api.use('npm-mongo', 'server');
+  api.use('npm-mongo@2.2.24', 'server');
```

Comment:
This seems important because the new options format won't work with older versions of `npm-mongo`.